apps/public-docsite-v9/.storybook/main.utils.js

```diff
@@ -6,6 +6,9 @@
 const fs = require('fs');
 const path = require('path');
 
+// Dependencies to exclude stories loading
+const excludedDependencies = ['@fluentui/react-overflow'];
```

Comment:
Terrible hack. but necessary for now until we have a better way of excluding package stories from storybook.